portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,31 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>My Portfolio UNDER CONSTRUCTION</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+</head>
+
+<body>
+    <nav id="top-nav-bar">
+        <a href="/">Home</a>
+        <a href="/random.html">random generator</a>
+    </nav>
```

Comment:
See if you can find a way to break out this navbar into it's own file so you don't have to copy and paste it in each file. This way, you only have to update it once and encourage code reuse.

Reply:
Hmm, I may need to use a server side include for this. It looks like the server is statically serving the files.

Reply:
So I found a way to do server side includes using jsp; Week 3 introduces us to java servlets, and I discovered jsp by accident when looking up ways of embedding   java in html pages. I can always revert this latest commit and do it client side using js if there is a better way of doing this.

Reply:
I reverted and forked the last commit; I moved the nav bar to a js file.